modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/hypnogoggles.dm

```diff
@@ -102,13 +102,14 @@
 /datum/brain_trauma/induced_hypnosis/New(phrase)
```

Comment:
None of this should be in the brain trauma. I don't know why it's even operating in this system. Hell, it shouldn't be doing try catches. This really needs refactored.

Reply:
Later, not in that PR. We have plans on atomizing, respriting and refactoring lots of stuff